### ISSUE TITLE
Keep getting exception on program exit...

### ISSUE THREAD

**Issue by OriginalPenguin** [yahoo/redislite#76]
I have a program in Python 3.4 that uses Redislite 1.0.241.

About a third of the time when the program exits I get the following error:

```
Exception ignored in: <bound method StrictRedis.__del__ of     StrictRedis<ConnectionPool<UnixDomainSocketConnection<path=/tmp/tmpjgllngma/redis.socket,db=0>>>>
Traceback (most recent call last):
  File "lib/python3.4/site-packages/redislite/client.py", line 345, in __del__
  File "lib/python3.4/site-packages/redislite/client.py", line 106, in _cleanup
AttributeError: 'NoneType' object has no attribute 'debug'
```

Any idea what's gone wrong? 

Thanks

**Comment by dwighthubbard** [comment]
Not sure what is going on.  A few pieces of information that would be helpful.

Which Python3 interpreter?  Python/cpython or pypy3?

How did you install the module?

What is the output of the following command:
    python -m redislite.debug

**Comment by dwighthubbard** [comment]
I submitted a fix that should fix this issue in redislite 1.0.243

**Comment by OriginalPenguin** [comment]
Awesome!  Thank you so much!
